Public/FourKeyMetrics.ps1

```diff
@@ -261,7 +341,7 @@ function Get-BucketedMetricsForPeriod($releaseMetrics, $endDate) {
     }
 
     if ($failedreleaseCount -gt 0){
-        $mttrMeasures = $releaseMetrics | Where-Object { $_.IsFix } | ForEach-Object { $_.Interval.TotalHours } | Measure-Object -Average
+        $mttrMeasures = $releaseMetrics | Where-Object { $_.IsFix } | ForEach-Object { $_.FailureDuration.TotalHours } | Measure-Object -Average
```

Comment:
Here's the problem that's causing [zero MTTR for reports that aren't merging a psuedo repo](https://github.com/red-gate/RedGate.Metrics/pull/24#issuecomment-2180166544).

We're pulling the `FailureDuration` from marked fix releases. That's being set to 0 for the fix release itself, on the assumption the fix release is the _end_ of a failure.

Reply:
Another option: I don;t think we need to process the from the most recent release, moving backwards. Pretty sure that's accidental implementation detail, rather than required.

I just did a crude local spike to do this analysis from the oldest release to the newest, and it _looks_ like it works OK.

Do you think the extra implementation/testing would be worthwhile to (hopefully) maintain the behaviour you're after?

If so I can patch my spike onto your branch for some more testing.